src/include/sof/audio/volume.h

```diff
@@ -157,6 +157,7 @@ struct vol_data {
 	vol_scale_func scale_vol;		/**< volume processing function */
 	vol_zc_func zc_get;			/**< function getting nearest zero crossing frame */
 	vol_ramp_func ramp_func;		/**< function for ramp shape */
+	bool copy_needed;			/**< control copy gain or not */
```

Comment:
let's name this `copy_gain` directly, because `copy_needed` is ambiguous, it's hard to know what is copied from the name.

Reply:
Yep, please change the name, copy_needed is confusing